modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/js/deleteCommerceOrdersPropsTransformer.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {fetch, openToast, postForm, sub} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {namespace},
+	formName,
+	selectedItemsKey,
+	...otherProps
+}) {
+	const handleDelete = (
+		confirm,
+		{
+			actions: {
+				delete: {href},
+			},
+			id,
+		},
+		loadData
+	) => {
+		if (confirm) {
+			fetch(href.replace('{id}', id), {method: 'DELETE'})
+				.then(({ok}) => {
+					if (ok) {
+						loadData();
+						openToast({
+							message: Liferay.Language.get(
+								'your-request-completed-successfully'
+							),
+							type: 'success',
+						});
+					}
+					else {
+						throw new Error();
+					}
+				})
+				.catch(() => {
+					openToast({
+						message: Liferay.Language.get(
+							'an-unexpected-error-occurred'
+						),
+						type: 'danger',
+					});
+				});
+		}
+	};
+
+	const handleBulkDelete = (action, confirm, keyValues) => {
+		if (confirm) {
+			const form = document.getElementById(`${namespace}${formName}`);
+
+			if (form) {
+				postForm(form, {
+					data: {
+						...action.data,
+						[`${selectedItemsKey}`]: keyValues.join(','),
+					},
+					url: action.href,
+				});
+			}
+			else {
+				openToast({
+					message: Liferay.Language.get(
+						'an-unexpected-error-occurred'
+					),
+					type: 'danger',
+				});
+			}
+		}
+	};
+
+	return {
+		...otherProps,
+		onActionDropdownItemClick({action, itemData, loadData}) {
+			if (action.data.id === 'delete') {
+				Liferay.Util.openConfirmModal({
+					message: `${sub(
+						Liferay.Language.get(
+							'are-you-sure-you-want-to-delete-order-x'
+						),
+						itemData.id
+					)}\n${Liferay.Language.get(
+						'this-operation-cannot-be-undone'
+					)}`,
+					onConfirm: (confirm) =>
+						handleDelete(confirm, itemData, loadData),
+					title: sub(
+						Liferay.Language.get('delete-order-x'),
+						itemData.id
+					),
+				});
+			}
+		},
+		onBulkActionItemClick({action, selectedData: {keyValues}}) {
+			if (action.data.id === 'delete') {
+				Liferay.Util.openConfirmModal({
```

Comment:
Import `openConfirmModal` from `frontend-js-web` instead of using it from the global object.